include/openssl/obj_mac.h

```diff
@@ -2839,15 +2839,157 @@
 #define NID_sha224              675
 #define OBJ_sha224              OBJ_nist_hashalgs,4L
 
-#define OBJ_dsa_with_sha2               OBJ_nistAlgorithms,3L
-
-#define SN_dsa_with_SHA224              "dsa_with_SHA224"
```

Comment:
So it seems you changed the SN from "dsa_with_SHA224" to "id-dsa-with-sha224". I assume because we didn't have an SN before. I'm wondering if this is going to break something.